curriculum/challenges/english/25-front-end-development/lab-book-organizer/67172b43f84bcd2dec238a3d.md

```diff
@@ -0,0 +1,152 @@
+---
+id: 67172b43f84bcd2dec238a3d
+title: Build a Book Organizer
+challengeType: 14
+dashedName: build-a-book-organizer
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should have an array of objects named `books` where each object in the array should have a string `title`, another string `authorName`, and a number `releaseYear`. 
+
+2. Your `books` array should have a minimum of three objects.
+
+3. You should have a callback function named `sortByYear` that accepts two books, `book1`, and `book2` as parameter for sorting the array.
+
+4. The `sortByYear` function should return `-1` if the `releaseYear` of `book1` object is smaller than that of the `book2` object.
+
+5. The `sortByYear` function should return `1` if the `releaseYear` of `book1` object is bigger than that of the `book2` object.
+
+6. The `sortByYear` function should return `0` for all other scenarios.
```

Comment:
What are the other scenarios? The two release years being equal?

Reply:
Yes. Should I mention that instead of "other scenarios"?

Reply:
It would be better probably :+1: